otherlibs/systhreads/st_posix.h

```diff
@@ -32,9 +32,9 @@
 #endif
 
 #ifdef __GNUC__
-#define INLINE inline
+#define INLINE Caml_inline
 #else
-#define INLINE
+#define INLINE static
```

Comment:
Since `Caml_inline` is available on all platforms, I would simply replace `INLINE` by `Caml_inline` in this file and delete this `#ifdef` nonsense.

Reply:
I wasn't totally sure about that, because `inline` was only used for GCC, but I guess that could have been dropped when we introduced the C99 requirement. I pushed a commit tidying it up.